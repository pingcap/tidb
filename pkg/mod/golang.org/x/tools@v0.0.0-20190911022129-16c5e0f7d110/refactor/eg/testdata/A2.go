// +build ignore

package A2

// This refactoring causes addition of "errors" import.
// TODO(adonovan): fix: it should also remove "fmt".

import myfmt "fmt"

func example(n int) {
	myfmt.Errorf("%s", "")
}
