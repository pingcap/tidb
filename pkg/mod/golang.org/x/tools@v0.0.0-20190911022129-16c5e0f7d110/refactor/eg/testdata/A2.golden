// +build ignore

package A2

// This refactoring causes addition of "errors" import.
// TODO(adonovan): fix: it should also remove "fmt".

import (
	"errors"
	myfmt "fmt"
)

func example(n int) {
	errors.New("")
}
